stream-compositions/services/legal-entity-composition-service/pom.xml

```diff
@@ -0,0 +1,153 @@
+                            <goal>repackage</goal>
+                        </goals>
+                        <configuration combine.self="override">
+                            <attach>false</attach>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Unpack OpenAPI specs from zip dependencies -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.backbase.stream.compositions.api</groupId>
+                                    <artifactId>legal-entity-composition-api</artifactId>
+                                    <version>${project.version}</version>
+                                    <classifier>api</classifier>
+                                    <outputDirectory>${project.build.directory}/yaml</outputDirectory>
+                                    <type>zip</type>
+                                    <overWrite>true</overWrite>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.backbase.stream.compositions</groupId>
+                                    <artifactId>legal-entity-integration-api</artifactId>
+                                    <version>${project.version}</version>
+                                    <classifier>api</classifier>
+                                    <type>zip</type>
+                                    <outputDirectory>${project.build.directory}/yaml</outputDirectory>
+                                    <overWrite>true</overWrite>
+                                </artifactItem>
+                            </artifactItems>
+                            <includes>**/*.yaml, **/*.json</includes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- Generate models and clients from unpacked OpenAPI specs -->
+            <plugin>
+                <groupId>com.backbase.oss</groupId>
+                <artifactId>boat-maven-plugin</artifactId>
+                <version>${boat-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>generate-config-service-api-code</id>
+                        <goals>
+                            <goal>generate-spring-boot-embedded</goal>
```

Comment:
Stream is entirely built with reactive components, and as we are also using the SSDK reactive starter lets also generate creative controller, you can use this guy as reference: https://github.com/Backbase/stream-services-2.0/blob/master/stream-legal-entity/legal-entity-http/pom.xml#L71

Reply:
but this example has security completely off, right ?